specification/repository.md

```diff
@@ -81,9 +81,14 @@ approval is granted, GDI repositories MUST NOT cut a GA release.
     changes that have not yet been released.
   - The file MUST be in reverse chronological order, with the most recent
     releases at the top of the file, after the `Unreleased` section.
+  - Each release SHOULD contain a list of changes from upstream that Splunk has
+    been working on, are relevant to Splunk GDI, or fix outstanding bugs.
+  - Each release MUST contain a link to the upstream release notes.
   - Each release SHOULD be separated by a line separator (`---`) from the other relases.
   - Each release SHOULD contain separate sections for each major functionality
     area (if applicable).
+  - Each change MUST bear a label that indicates where the change is coming from.
```

Comment:
I am not convinced if we need this (especially as `MUST`). 

1. For example if there is a security fix in a defendant library then should we list it as well ?
2. Cannot the user simply look at the changelog of the dependency? E.g. in .NET distro the OTel release is listed in the changelog https://github.com/signalfx/splunk-otel-dotnet/blob/main/CHANGELOG.md . Do you prefer to copy-paste all release notes from ALL OTel packages? It would take a lot of effort to do it.

Reply:
This is addressed by

>   - Each release SHOULD contain a list of changes from upstream that Splunk has
    been working on, are relevant to Splunk GDI, or fix outstanding bugs.

So what to include is still discretionary.

Reply:
I see, but then one could have `(Splunk)` in each changelog entry which would look very awkward.

Reply:
We could omit the label when it's (Splunk), as it's sort of implied, yes.

Reply:
We could omit the label 

Right now the PR does not allow it 

> Each change MUST bear a label that indicates where the change is coming from.

Reply:
Fixin'!

Reply:
PTAL